test/ProjectService.Mappers.UnitTests/Responses/TaskResponseMapperTests.cs

```diff
@@ -0,0 +1,185 @@
+using System;
+using System.Collections.Generic;
+using LT.DigitalOffice.Models.Broker.Models;
+using LT.DigitalOffice.ProjectService.Mappers.Interfaces;
+using LT.DigitalOffice.ProjectService.Mappers.Models;
+using LT.DigitalOffice.ProjectService.Mappers.Models.Interfaces;
+using LT.DigitalOffice.ProjectService.Mappers.Responses;
+using LT.DigitalOffice.ProjectService.Mappers.Responses.Interfaces;
+using LT.DigitalOffice.ProjectService.Models.Db;
+using LT.DigitalOffice.ProjectService.Models.Dto.Models;
+using LT.DigitalOffice.ProjectService.Models.Dto.Models.ProjectUser;
+using LT.DigitalOffice.ProjectService.Models.Dto.Responses;
+using LT.DigitalOffice.UnitTestKernel;
+using NUnit.Framework;
+
+namespace LT.DigitalOffice.ProjectService.Mappers.UnitTests.Responses
+{
+    public class TaskResponseMapperTests
+    {
+        private DbTask _dbTask;
+        private TaskResponse _taskResponse;
+        private IProjectInfoMapper _projectInfoMapper = new ProjectInfoMapper();
+        private ITaskInfoMapper _taskInfoMapper = new TaskInfoMapper();
+        private ITaskPropertyInfoMapper _taskPropertyInfoMapper = new TaskPropertyInfoMapper();
+        private IProjectUserInfoMapper _projectUserInfoMapper = new ProjectUserInfoMapper();
+
+        private ITaskResponseMapper _mapper;
+
+        private readonly Guid _taskId = Guid.NewGuid();
+        private readonly string _name = "Name";
+        private readonly string _description = "Description";
+        private readonly int _number = 1;
+        private readonly int _plannedMinutes = 60;
+        private readonly DateTime _createdAt = DateTime.UtcNow;
+        private Guid _userId = Guid.NewGuid();
+        private string _firstName = "Name";
+        private string _middleName = "Middle Name";
+        private string _lastName = "Last Name";
+        private bool _isActive = true;
+
+        private UserData _authorUserData;
+        private UserData _parentUserData;
+        private UserData _parentAssignedUserData;
+        private UserData _parentAuthorAssignedUserData;
+        private readonly string _departmentName = "Department name";
```

Comment:
![image](https://user-images.githubusercontent.com/38541098/124888527-dab59880-dfde-11eb-8095-2bd87ac5b5f0.png)

Reply:
They are not necessary, but when somebody will do tests, it will be valueable